include/sph/findNeighborsSfc.hpp

```diff
@@ -57,7 +57,7 @@ size_t neighborsSumImpl(const Task& t)
     size_t sum = 0;
 
     #pragma omp parallel for reduction(+ : sum)
-    for (unsigned int i = 0; i < t.size(); i++)
+    for (size_t i = 0; i < (size_t)t.size(); i++)
```

Comment:
It would be better to change the return type to `size_t` in Task.hpp

Reply:
Perfect. I removed the cast and I changed the Task.size() function. The commit is in-place.